website/identifiers/metadata.py

```diff
@@ -57,3 +63,60 @@ def format_contrib(contributor):
         publication_year=getattr(node.registered_date or node.date_created, 'year'),
         pretty_print=pretty_print
     )
+
+
+def format_creators(preprint):
+    creators = []
+    for contributor in preprint.node.visible_contributors:
+        creator = CREATOR(E.creatorName(u'{}, {}'.format(contributor.family_name, contributor.given_name)))
+        creator.append(E.givenName(contributor.given_name))
+        creator.append(E.familyName(contributor.family_name))
+
+        # contributor.external_identity = {'ORCID': {'1234-1234-1234-1234': 'VERIFIED'}}
+        if contributor.external_identity.get('ORCID'):
+            orcid_value = contributor.external_identity['ORCID'].keys()[0]
+            if contributor.external_identity['ORCID'][orcid_value] == 'VERIFIED':
+                creator.append(E.nameIdentifier(orcid_value, nameIdentifierScheme='ORCID', schemeURI='http://orcid.org/'))
+
+        creators.append(creator)
+
+    return creators
+
+
+def format_subjects(preprint):
+    subject_names = set()
```

Comment:
I couldn't find any info at all about order of subjects on the [datacite metadata schema PDF](https://schema.datacite.org/meta/kernel-4.0/doc/DataCite-MetadataKernel_v4.0.pdf), so I'm guessing not? I could be wrong though.